hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestDataGenerator.java

```diff
@@ -578,8 +576,8 @@ private static void createMetadataFile(String f, String basePath, StorageConfigu
         basePath + "/" + HoodieTableMetaClient.METAFOLDER_NAME + "/" + f);
     OutputStream os = null;
     try {
-      FileSystem fs = HadoopFSUtils.getFs(basePath, configuration);
-      os = fs.create(commitFile, true);
+      HoodieStorage storage = HoodieStorageUtils.getStorage(basePath, configuration);
```

Comment:
Changes here can be in an independent PR.

Reply:
HadoopFSUtils is removed so it was needed here